src/app/containers/ChartbeatAnalytics/utils/index.js

```diff
@@ -133,7 +133,9 @@ export const getConfig = ({
   chartbeatDomain,
   mostReadTitle,
 }) => {
-  const referrer = getReferrer(platform, origin, previousPath);
+  const referrer =
+    previousPath || isAmp ? getReferrer(platform, origin, previousPath) : null;
```

Comment:
I don't really understand how the result of `getReferrer` is used here: https://github.com/bbc/simorgh/blob/latest/src/app/lib/analyticsUtils/index.js#L121-L123 

It seems to return an escaped template string: `\${documentReferrer}`, where is that resolved? I'd just like to know this so I can see whether or not document referrer contains the external referrer or not for the purposes of this bug?

I looks correct for canonical, I feel there should be a test somewhere for the amp case.

Reply:
@andrewscfc [that's an amp analytics variable](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#document-referrer) used to compute a `document.referrer` equivalent for amp pages since the window `document` on amp would be undefined.

> where is that resolved?

if you check out this url http://localhost:7080/news/articles/c6v11qzyv8po.amp on amp, it resolves to the `v` value in the chartbeat pings.

Reply:
Thanks @rhenshaw56 , I can see the test here now I understand. I wonder if it's worth a comment to explain that template string, it doesn't seem very obvious _to me_ (may just be me!). I've found the test here btw https://github.com/bbc/simorgh/blob/be825ee361165266b2e70a0a333ad40b1bc45f4f/src/app/containers/ChartbeatAnalytics/utils/index.test.js#L387

For this bug, will `amp-analytics` populate that value with external referrers or just internal referrers? We obviously don't want it to populate `v` for external referrers.

Reply:
> For this bug, will amp-analytics populate that value with external referrers or just internal referrers? We obviously don't want it to populate v for external referrers

@andrewscfc  I tested this out for amp, and just internal referrers are populated.

>  I wonder if it's worth a comment to explain that template string
Sure, I'll do that now.

Reply:
@andrewscfc done.